src/AcceptanceTests/Routing/When_using_single_namespace.cs

```diff
@@ -63,8 +37,13 @@ public class SourceEndpoint : EndpointConfigurationBuilder
         {
             public SourceEndpoint()
             {
-                EndpointSetup<DefaultServer>(endpointConfiguration =>
-                        endpointConfiguration.ConfigureTransport().Routing().RouteToEndpoint(typeof(MyRequest), typeof(TargetEndpoint)));
+                EndpointSetup<DefaultServer>(c =>
+                {
+                    var transport = c.ConfigureAzureServiceBus();
```

Comment:
How did you prevent the automatic invocation of this method next to this explicit invocation ?

Reply:
I see this just exposes our config, what I really meant is how are this config and ConfigureEndpointAzureServiceBusTransport not overwriting each other

Reply:
Not sure I follow @yvesgoeleven

Reply:
Explained in zoom, it was more a question of me not understanding the order of invocation